features/util.py

```diff
@@ -88,38 +88,61 @@ def lxc_get_property(name: str, property_name: str, image: bool = False):
         return value
 
 
-def repo_state_hash(include_features: bool = False):
+def repo_state_hash(
+    exclude_dirs: List[str] = [".github", "demo", "features", "sru", "tools"]
```

Comment:
Even though we are not doing this at the moment, but if we modify the `exclude_dirs` default list on the function, we will modify it on all other calls of `repo_state_hash`.

I know we are not doing that, but it is better to be defensive here. It is better to have that as `None`  by default and set it up with a default list.